demos/modipulate-gml/test/main.c

```diff
@@ -0,0 +1,200 @@
+/* Demo test app for modipulate-gml */
+
+#include <stdio.h>
+#include "modipulate_gml.h"
+
+#ifdef _WIN32
+#include <windows.h>
```

Comment:
Let's make sure we set the lean and mean macro before including windows.h.  More info: https://support.microsoft.com/en-us/kb/166474